qubesadmin/tools/qvm_backup_restore.py

```diff
@@ -220,9 +220,8 @@ def main(args=None, app=None):
         # unreachable - error_runtime will raise SystemExit
         return 1
 
-    if args.ignore_missing:
-        backup.options.use_default_template = True
-        backup.options.use_default_netvm = True
+    backup.options.use_default_template = \
+    backup.options.use_default_netvm = args.ignore_missing
```

Comment:
Code style: either correctly indent line continuation, or assign `args.ignore_missing` to both separately.

Reply:
> or assign `args.ignore_missing` to both separately.

Done.